x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/text_expansion_callout.tsx

```diff
@@ -333,6 +338,19 @@ export const TextExpansionCallOut: React.FC<TextExpansionCallOutProps> = (props)
     isModelStarted,
     isStartButtonDisabled,
   } = useValues(TextExpansionCalloutLogic);
+  const { error: createError } = useValues(CreateTextExpansionModelApiLogic);
```

Comment:
Suggestion: I would access the error through the `TextExpansionCalloutLogic` than through `CreateTextExpansionModelApiLogic` directly.

Reply:
I agree, when we need to put the error at the top of the page we wanted to use the API logic directly so that we didn't start the polling. But now that we changed to update the panel we can get the errors from the `TextExpansionCalloutLogic`.

Reply:
Good idea - I've updated the values interface of `TextExpansionCalloutLogic` and the code to get the errors from there.